src/main/java/manager/DatabaseLoader.java

```diff
@@ -63,155 +76,166 @@
+                armor.setTyp(getString(armorSet, getLocalized("column.type")));
+                armor.setSubTyp(getString(armorSet, getLocalized("column.subtype")));
+                armor.setRequirement(getString(armorSet, getLocalized("column.requirement")));
+                armor.setProtection(armorSet.getInt(getLocalized("column.protection")));
+                armor.setWeight(armorSet.getDouble(getLocalized("column.weight")));
+                armor.setEffect(getString(armorSet, getLocalized("column.effect")));
+                armor.setSlots(armorSet.getInt(getLocalized("column.upgradeSlots")));
+                armor.setRarity(getRarity(armorSet, getLocalized("column.rarity")));
+                armor.setCost(getString(armorSet, getLocalized("column.price")));
+                armor.setTier(armorSet.getInt(getLocalized("column.tier")));
 
                 armorList.add(armor);
 
-                TypTranslation.add(armor.getTyp(), "Rüstung");
+                TypTranslation.add(armor.getTyp(), getLocalized("type.armor"));
             }
 
+        } catch (SQLException e) {
+            return getErrorString("table.armors");
+        } finally {
             Platform.runLater(() -> {
                 Database.armorList.set(armorList);
                 semaphore.release();
             });
-        } catch (SQLException e) {
-            return "Rüstungen konnten nicht geladen werden.";
         }
         return "";
     }
 
     private static String loadJewellery(Statement statement, Semaphore semaphore) {
-        try (ResultSet jewellerySet = statement.executeQuery("SELECT * FROM Items NATURAL JOIN Schmuck")) {
-            ObservableList<Jewellery> jewelleryList = FXCollections.observableArrayList();
+        ObservableList<Jewellery> jewelleryList = FXCollections.observableArrayList();
+
+        try (ResultSet jewellerySet = statement.executeQuery(format("SELECT * FROM %s NATURAL JOIN %s", "table.items", "table.jewellery"))) {
 
             while (jewellerySet.next()) {
                 Jewellery jewellery = new Jewellery();
-                jewellery.setName(getString(jewellerySet, "Bezeichnung"));
-                jewellery.setMaterial(getString(jewellerySet, "Material"));
-                jewellery.setGem(getString(jewellerySet, "Edelstein"));
-                jewellery.setTyp(getString(jewellerySet, "Typ"));
-                jewellery.setSubTyp(getString(jewellerySet, "Subtyp"));
-                jewellery.setRequirement(getString(jewellerySet, "Vorraussetzung"));
-                jewellery.setEffect(getString(jewellerySet, "Effekt"));
-                jewellery.setSlots(jewellerySet.getInt("Verbesserungsslots"));
-                jewellery.setRarity(getString(jewellerySet, "Seltenheit"));
-                jewellery.setCost(getString(jewellerySet, "Preis"));
-                jewellery.setTier(jewellerySet.getInt("Tier"));
+                jewellery.setName(getString(jewellerySet, getLocalized("column.name")));
```

Comment:
same here

Reply:
A consistent naming scheme is easier to maintain.
In this way I force the user to use one.